src/main/resources/cubicchunks.accesswidener

```diff
@@ -11,4 +11,5 @@ accessible class net/minecraft/world/level/levelgen/VerticalAnchor$BelowTop
 accessible class net/minecraft/world/level/levelgen/VerticalAnchor$AboveBottom
 accessible class net/minecraft/world/level/levelgen/VerticalAnchor$Absolute
 extendable class net/minecraft/server/level/Ticket
-accessible method net/minecraft/server/level/ChunkMap$DistanceManager <init> (Lnet/minecraft/server/level/ChunkMap;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)V
\ No newline at end of file
+accessible method net/minecraft/server/level/ChunkMap$DistanceManager <init> (Lnet/minecraft/server/level/ChunkMap;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)V
+accessible method net/minecraft/world/level/chunk/storage/IOWorker loadAsync (Lnet/minecraft/world/level/ChunkPos;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
Avoid AccessWideners/transformers if possible. Use accessor mixins instead (note for us: the constructor AW should also be replaced)